pkg/sriov/sriov.go

```diff
@@ -247,6 +248,18 @@ func (s *sriovManager) ApplyVFConfig(conf *types.NetConf) error {
 		if err := s.setVfGUID(conf, pfLink, conf.GUID); err != nil {
 			return err
 		}
+	} else {
+		// Verify VF have valid GUID.
+		vfLink, err := s.nLink.LinkByName(conf.HostIFNames)
+		if err != nil {
+			return fmt.Errorf("failed to lookup vf %q: %v", conf.HostIFNames, err)
+		}
+
+		hwAddr := vfLink.Attrs().HardwareAddr.String()
+		match, _ := regexp.MatchString("((00:){7}00|(ff:){7}ff|(FF:){7}FF)$", hwAddr)
```

Comment:
can we have something like `utils.IsValidGUID(addr net.HardwareAddr)` ?

Reply:
or just extend `utils.IsValidGUID` with checking all ff guid as well and then use the method here

Reply:
> can we have something like `utils.IsValidGUID(addr net.HardwareAddr)` ?
I think, it'll look more confusing then hwAddr[36:]. For me, hwAddr[36:] even more readable, since it easily distinguishable over allFancyFunctionNames.
> or just extend utils.IsValidGUID with checking all ff guid as well and then use the method here
Sound good.